accounts/usbwallet/ledger.go

```diff
@@ -47,15 +47,28 @@ type ledgerParam1 byte
 // specific opcodes. The same parameter values may be reused between opcodes.
 type ledgerParam2 byte
 
+// appType is used to identify which app is being used on the device
+type appType byte
+
 const (
 	ledgerOpRetrieveAddress  ledgerOpcode = 0x02 // Returns the public key and Ethereum address for a given BIP 32 path
 	ledgerOpSignTransaction  ledgerOpcode = 0x04 // Signs an Ethereum transaction after having the user validate the parameters
 	ledgerOpGetConfiguration ledgerOpcode = 0x06 // Returns specific wallet application configuration
 
+	ledgerOpSignHashBLS ledgerOpcode = 0x02 // Signs hash for BLS validator app
+
+	ledgerOpGetPubkeyBLS ledgerOpcode = 0x04 // Returns public key for BLS validator app
+
+	ledgerP1UnfinishedBLSData ledgerParam1 = 0x00 // BLS message/hash data to be signed, not finished 
```

Comment:
can you explain why this is needed?

Reply:
It is not needed. Holdover from the original signer app I used that seemed easier to leave in than take out